app/src/main/java/com/github/HumanLearning2021/HumanLearningApp/offline/CachePictureStorage.kt

```diff
@@ -0,0 +1,14 @@
+package com.github.HumanLearning2021.HumanLearningApp.offline
+
+import android.content.Context
+import dagger.hilt.android.qualifiers.ApplicationContext
+import javax.inject.Inject
+
+/**
+ * PictureStorage whose save location is the application cache
+ */
+class CachePictureStorage @Inject constructor(
+    dbName: String,
+    @ApplicationContext context: Context
+) :
+    PictureStorage(dbName, context, context.cacheDir)
```

Comment:
```suggestion
fun CachePictureStorage(
    dbName: String,
    context: Context
) =
    PictureStorage(dbName, context, context.cacheDir)
```
This should be a factory function. Also see naming comment in superclass

Reply:
I will put it into the companion object of `PictureCache`